addons/udes_delivery_control/models/stock_picking.py

```diff
@@ -267,7 +361,7 @@ def action_delivery_control_done(self):
         for picking in self.filtered("u_is_delivery_control"):
             picking.write({"state": "done", "date_done": fields.Datetime.now()})
 
-            if not picking.u_goods_in_picking_id:
+            if not picking.u_goods_in_picking_id and (picking.u_loading_type != "backload"):
```

Comment:
Nit: Don't need brackets.

Reply:
Maybe`picking.u_is_unload` would be neater here too?

Reply:
done

Reply:
Wrong field was used here, have raised a separate comment: https://github.com/unipartdigital/udes-open/pull/394#discussion_r533487629

Reply:
done